src/test/ReactTestUtils.js

```diff
@@ -470,6 +470,11 @@ ReactShallowRenderer.prototype._render = function(element, transaction, context)
 function makeSimulator(eventType) {
   return function(domComponentOrNode, eventData) {
     var node;
+    invariant(
+      !React.isValidElement(domComponentOrNode),
+      'TestUtils.Simulate expects a component instance and not a React Element.' +
```

Comment:
nit: Can you write "ReactElement" without a space? That's consistent with our prop types message.